src/game-logic/preGame.js

```diff
@@ -0,0 +1,61 @@
+import cardRoundLogic from './card-logic/cardRoundLogic';
+
+/**
+ * Pre-game logic that occurs before the spell-card game begins
+ */
+const TIMERLENGTH = 30000;
+
+export default function pregame(socket, roomID) {
+   // start timer in socket.io room
+   socket.to(roomID).emit('start pre-game');
```

Comment:
Not sure what the best practices are around this, but do we want the socket events to be kept as constants rather than strings?

Reply:
yeah constants seem to be a good idea. I'll change that up.